README.md

```diff
@@ -1,12 +1,27 @@
-# Tsickle - TypeScript to Closure Annotator [![Build Status](https://travis-ci.org/angular/tsickle.svg?branch=master)](https://travis-ci.org/angular/tsickle)
+# Tsickle - TypeScript to Closure Translator [![Build Status](https://travis-ci.org/angular/tsickle.svg?branch=master)](https://travis-ci.org/angular/tsickle)
 
-Tsickle processes TypeScript and adds [Closure Compiler]-compatible JSDoc
-annotations. This allows using TypeScript to transpile your sources, and then
+Tsickle converts TypeScript code into a form acceptible to the [Closure Compiler].
+This allows using TypeScript to transpile your sources, and then using
 Closure Compiler to bundle and optimize them, while taking advantage of type
 information in Closure Compiler.
 
 [Closure Compiler]: https://github.com/google/closure-compiler/
 
+## What conversion means
+
+A (non-exhaustive) list of the sorts of transformations Tsickle applies:
+
+- inserts Closure-compatible JSDoc annotations on functions/classes/etc
+- converts ES6 modules into `goog.module`/`goog.provide`
```

Comment:
So far in my experience (keeping in mind that I'm probably doing many things wrong, but in my defense there is not an abundance of documentation...) I have needed to be aware of how these module systems interoperate in Closure. By the way the documentation on that last bit is not particularly encouraging:

https://github.com/google/closure-compiler/wiki/JS-Modules#module-interoperation

However I am completely thrilled that making it moot is a goal - In this readme is a great improvement regardless of my nitpick :-)